admin/commands/common/set_profiler_enabled.go

```diff
@@ -0,0 +1,37 @@
+package common
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	"github.com/onflow/flow-go/admin"
+	"github.com/onflow/flow-go/admin/commands"
+	"github.com/onflow/flow-go/utils/debug"
+)
+
+var _ commands.AdminCommand = (*SetProfilerEnabledCommand)(nil)
+
+type SetProfilerEnabledCommand struct{}
+
+func (s *SetProfilerEnabledCommand) Handler(ctx context.Context, req *admin.CommandRequest) (interface{}, error) {
+	enabled := req.ValidatorData.(bool)
+	debug.SetProfilerEnabled(enabled)
+	return "ok", nil
+}
+
+func (s *SetProfilerEnabledCommand) Validator(req *admin.CommandRequest) error {
+	enabled, ok := req.Data.(string)
+	if !ok {
+		return errors.New("the input must be true or false")
+	}
+	if enabled == "true" {
```

Comment:
i guess we don't think it's worth doing strings.EqualFold?

Reply:
I'd like to keep it simple and stupid.

Reply:
We actually don't need to use a string at all, booleans are natively supported in JSON. So I'd suggest instead using a bool:

```
enabled, ok := req.Data.(bool)
```